apps/editors/templates/editors/themes/themes.html

```diff
@@ -42,7 +42,9 @@
     </dl>
     <dl class="meta">
       {% if hasOneToOne(theme, 'themelock') %}
-        <dt>{{ _('Reviewer') }}</dt><dd>{{ emaillink(theme.themelock.reviewer.email) }}</dd>
+        {% if theme.themelock.reviewer != user %}
```

Comment:
I think this is going to need to be `request.user` rather than just `user`. Can you put these in a single `if` statement?